include/wget/wget.h

```diff
@@ -265,6 +265,8 @@ int
 	wget_global_get_int(int key) LIBWGET_EXPORT;
 void
 	(*wget_global_get_func(int key))(const char *, size_t) LIBWGET_EXPORT;
+void
+	wget_handle_sigwinch(void) LIBWGET_EXPORT;
```

Comment:
Almost done... I just cross-compiled your commits - SIGWINCH is unknown :-)))
```
wget.c:839:20: error: 'SIGWINCH' undeclared (first use in this function)
  } else if (sig == SIGWINCH) {
                    ^~~~~~~~
```
Not sure about gnulib... could you fix that and than I'll merge your changes (likely tomorrow morning).